OmniKit/MessageBlocks/ConfigureAlertsCommand.swift

```diff
@@ -95,7 +95,8 @@ public struct ConfigureAlertsCommand : MessageBlock {
                 let minutes = UInt16(duration.minutes)
                 data.appendBigEndian(minutes)
             }
-            data.appendBigEndian(UInt16(beepType.rawValue) << 8 + UInt16(beepRepeat))
+            data.append(UInt8(beepType.rawValue))
+            data.append(UInt8(beepRepeat))
```

Comment:
If you change beepRepeat to a UInt8, then you don't need to worry about casting, and possible errors caused by that (if you cast a number > 255 into a UInt8 you'll get an exception)

Reply:
Yes, I was unsure where it is essential to explicitly declare the cast type.